sdks/java/core/src/main/java/org/apache/beam/sdk/values/PCollectionViews.java

```diff
@@ -233,7 +233,7 @@ public T apply(Iterable<WindowedValue<T>> contents) {
 
     @Override
     public Iterable<T> apply(Iterable<WindowedValue<T>> contents) {
-      return Iterables.unmodifiableIterable(
+      return ImmutableList.copyOf(
```

Comment:
Reverted. It is just a temporary workaround for a deserialization problem which has been fixed now.